syncmaster/backend/dependencies/stub.py

```diff
@@ -0,0 +1,53 @@
+# SPDX-FileCopyrightText: 2023-2024 MTS PJSC
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: 2023 Andrey Tikhonov (@Tishka17)
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
```suggestion
# SPDX-FileCopyrightText: 2023 Andrey Tikhonov (@Tishka17)
# SPDX-License-Identifier: Apache-2.0
```
Add this file to [exclude](https://github.com/MobileTeleSystems/horizon/blob/1.0.1/.pre-commit-config.yaml#L53)

Reply:
Comment is resolved, but duplicated file header is still there